data/brands/shop/video_games.json

```diff
@@ -48,6 +48,21 @@
         "shop": "video_games"
       }
     },
+    {
+      "displayName": "EB Games",
```

Comment:
Sorry, the current displayName requirement for each entry is unique, so now there are two "EB Games" will cause an error

I suggest you change them to distinctive names such as "EB Games (Canada)" and "EB Games (AU & NZ)" respectively.

Reply:
I've just updated them both.